docs/concepts/core-service/README.mdx

```diff
@@ -66,44 +65,47 @@ If you want to directly call OIDC APIs, remember to set header `Content-Type: ap
 - [OpenID Connect Core](https://openid.net/specs/openid-connect-core-1_0.html)
 - [OpenID Connect Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)
 - [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
+- [OpenID Connect Back-Channel Logout](https://openid.net/specs/openid-connect-backchannel-1_0-final.html)
+- [OAuth 2.0](https://www.rfc-editor.org/rfc/rfc6749.html)
+- [OAuth 2.0 Token Introspection](https://www.rfc-editor.org/rfc/rfc7662.html)
 - [OAuth 2.0 Token Revocation](https://www.rfc-editor.org/rfc/rfc7009.html)
 - [OAuth 2.0 Resource Indicators](https://www.rfc-editor.org/rfc/rfc8707.html)
+- [Proof Key for Code Exchange (PKCE)](https://www.rfc-editor.org/rfc/rfc7636.html)
+
+## Public API
 
-## Management API
+### Management API
 
 _Management API_ is a set of APIs that manage and update Logto data. Only users with the `admin` role have access to them.
 
-It's mounted at `/api`. Head to [API references](https://openapi.logto.io/) to see the details in your browser, or use `GET /api/swagger.json` for a Swagger result in JSON.
+Head to [API references](https://bump.sh/logto/doc/logto-management-api) to see the details.
```

Comment:
We are still going to use the URL with `logto` in it. So let's keep using `https://openapi.logto.io`